test/access/AccessControl.behavior.js

```diff
@@ -639,6 +642,15 @@ function shouldBehaveLikeAccessControlDefaultAdminRules(errorPrefix, delay, defa
       );
     });
 
+    it('no op if renouncing when not having the role', async function () {
```

Comment:
Small NIT:

```suggestion
    it('no-ops if renouncing when not having the role', async function () {
```

I personally like to read the tests as `it('<verb> ...` (eg. `it('does something ...')`)